nerdlandbot/commands/GuildData.py

```diff
@@ -255,15 +261,60 @@ async def update_notification_audit(self, list_name: str) -> bool:
             return False
 
         notification_list = self.notification_lists[list_name]
-        
+
         if not "notified_on" in notification_list.keys():
             notification_list["notified_on"] = []
 
         notification_list["notified_on"].append(datetime.now().isoformat())
         await self.save()
 
         return True
+
+    async def add_pet(self, pet_name: str, user_id: str, category: str):
+        pet_id = await self.get_new_pet_id()
+        pet_id_str = str(pet_id)
+        self.pets[pet_id_str] = {}
+        self.pets[pet_id_str]['owner'] = user_id
+        self.pets[pet_id_str]['pet_name'] = pet_name.lower()
+        self.pets[pet_id_str]['category'] = category.lower()
+        await self.save()
         
+    
+
+    async def delete_pet(self, pet_id: str) -> bool:
+        pets = self.pets
+
+        del pets[pet_id]
```

Comment:
If you want to make use of the return value. You could check if the pet id exists and if not, return False. Though it's not even used, so I would perhaps just return None, but eh.